src/common/util/game_settings.hpp

```diff
@@ -38,6 +38,7 @@ const bool SHOW_PLAYER_IDS = true;
 /* Game Logic */
 const bool USE_INTERPOLATION_REPLAY = false;
 
+const int NAT_PUNCH_INTERVAL = 30;
```

Comment:
Do we really need to send this every 30ms? 
How do we know what to make this interval, what does it depend on? 

Would it work if we just did it once at the start of the connection and then never again?

Reply:
Oh just read your commit comment. 30sec sounds good

Btw I would use the sfml [clock](https://www.sfml-dev.org/documentation/2.5.1/classsf_1_1Clock.php) module instead of `<time.h>`. It has this nice feature where you can just get the elapsed time since the last time you called `clock.reset()` and its syntax makes the granularity of time very explicit e.g. `clock.getElapsedTime().asSeconds()`

Reply:
that's cool, I think I'll hold off on changing the code since it's fine and it's actually a different way of thinking about the problem to use a clock and elapsed time vs sampling the current time and keeping a timestamp. Rather spend my brainwaves on more core problems for Prototype 1